src/pages/documentation/mobile-core/configuration/api-reference.md

```diff
@@ -48,11 +46,7 @@ iOS
 
 ## configureWithFileInAssets
 
-<InlineAlert variant="info" slots="text"/>
-
-This API is only available in Android and was added in Android was added in Android Core version 1.7.0.
-
-You can bundle a JSON configuration file in the app's Assets folder to replace or complement the configuration that was downloaded by using the [Configure with App ID per environment](./index.md#configure-with-app-id-per-environment) approach.
+You can bundle a JSON configuration file in the app's Assets folder to replace or complement the configuration that was downloaded by using the [Configure with App ID per environment](../#configure-with-app-id-per-environment) approach.
```

Comment:
Actually just to confirm - is this a link within the Configuration overview page? There's some... wonkiness with relative links within tabs, which is why I want to confirm that before giving approval. 

(Basically, relative links within tabs are treated as links within the document itself, so the relative link won't work on VSCode, but it _does_ work on the site. It's weird.)

Reply:
Yes. This is a link in the Configuration overview page. I tested this running an instance locally and verified that it does resolve to the right location. I can also change this to `../index.md#configure-with-app-id-per-environment`  which is equivalent.

Reply:
Oooh okay - so long as you ran it locally and it works, that's all fine with me.